jsonschema-validation.xml

```diff
@@ -945,14 +947,21 @@
                     consumer than that which processed the containing document.
                 </t>
                 <t>
-                    All keywords in this section apply only to strings, and have no
-                    effect on other data types.
+                    All keywords in this section generally apply to strings, and have no
+                    effect on other JSON data types.  Additionally, they MAY be used without
+                    type information when describing resources of other media types, subject
+                    to certain restrictions.
                 </t>
                 <t>
                     Implementations MAY offer the ability to decode, parse, and/or validate
-                    the string contents automatically.  However, it MUST NOT perform these
+                    the string contents automatically.  However, they MUST NOT perform these
                     operations by default, and MUST provide the validation result of each
-                    string-encoded document separately from the enclosing document.  This
+                    string-encoded document separately from the enclosing document.  In particular,
+                    these keywords, including "contentSchema", MUST NOT cause the containing schema
+                    to fail validation.
+                </t>
+                <t>
+                    The optional automatic decoding, parsing, and validating
                     process SHOULD be equivalent to fully evaluating the instance against
                     the original schema, followed by using the annotations to decode, parse,
```

Comment:
Should this say instead, "The optional automatic decoding, parsing, and validating process SHOULD be equivalent to fully evaluating the instance against process SHOULD be equivalent to fully evaluating **an** instance against **a** schema, followed by using the annotations to decode, parse the original schema". The thing is, this section is about applying schemas or rules to encoded "content". if we use "evaluating **the** instance against **the** original schema", then it feels like there's confusion between the schema containing all this stuff vs. the stuff inside "content". Does this make sense?